.eslintrc

```diff
@@ -0,0 +1,87 @@
+{
```

Comment:
Good to merge. Fix is mostly whitespace, it's totally safe. I'm not really
into enforcing quote style.

Would you mind adding houndci as well once this is merged? I don't have
permissions. I'd just like to try it, maybe combined with github's setting
that disables the merge button when checks aren't passing.

On Friday, June 24, 2016, kyle shay <notifications@github.com> wrote:

> In .eslintrc
> <https://github.com/DestinyItemManager/DIM/pull/630#discussion_r68484071>:
>
> > @@ -0,0 +1,87 @@
> > +{
>
> 👍 nicely done! thank you!!
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/DestinyItemManager/DIM/pull/630/files/126c041f9f32413f2413dfcaacffa02b293882d9..7db5b9687f50b69ab0dfa8b5c1b194cbae667f9f#r68484071>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AATHeGzs1-6s3g0ABswGAz2b3lj7mvoUks5qPLMfgaJpZM4I8e0V>
> .
>